### PR TITLE
Test 4

### PR DESCRIPTION
**Pull request by Max-Korsakov**

### CODE REVIEW

File: src/groups/groups.service.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    private groupsModel: typeof groups,
+    @InjectModel(permissions)
+    private permissionsModel: typeof permissions,
+    @InjectModel(permissions_groups)
+    private permissionsGroupsModel: typeof permissions_groups,
+    private sequelize: Sequelize,
+  ) {}
+
+  async create(createGroupDto: CreateGroupDto) {
+    const t = await this.sequelize.transaction();
```
Comment (alex-sadovsky) [inline src/groups/groups.service.ts]: Could you please give to `t` more readable name?

File: src/groups/groups.service.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            },
+            { transaction: t },
+          );
+        }),
+      );
+      await t.commit();
+      return group;
+    } catch (error) {
+      await t.rollback();
+      console.log(error.message);
```
Comment (alex-sadovsky) [inline src/groups/groups.service.ts:56]: I think this is not a good practice to use `console.log` in the code. You can use NestJS logger service, or do not log it at all, but passthrough an error to the response

File: src/groups/groups.service.ts (hunk 1 above)
Comment (Max-Korsakov) [reply src/groups/groups.service.ts]: fixed

File: src/groups/groups.service.ts (hunk 2 above)
Comment (Max-Korsakov) [reply src/groups/groups.service.ts:56]: fixed

**Pull request merged**